demos/python_demos/multi_camera_multi_person_tracking/config.py

```diff
@@ -0,0 +1,14 @@
+time_window = 20
+global_match_thresh = 0.35
+
+sct_config = dict(
+    time_window=10,
+    continue_time_thresh=2,
+    track_clear_thresh=3000,
+    match_threshold=0.375,
+    merge_thresh=0.3,
+    n_clusters=4,
+    max_bbox_velocity=0.2,
+    detection_occlusion_thresh=0.7,
+    track_detection_iou_thresh=0.5
+)
```

Comment:
I like the idea of providing suitable option values, but I think it should be specified (foe example just in comments) for which models this config was written because there is a number of detection and reid nets.

Reply:
Description has been added.

Reply:
By `a number of detection and reid nets` I meant person-detection-retail-???? and person-reidentification-retail-????. The question was it this config fine tuned for a specific model.

Reply:
Actually this config is suitable for all these models. It should be changed only if 3rd party reid and detection models are used.